lib/ReactGridLayout.jsx

```diff
@@ -348,6 +376,15 @@ export default class ReactGridLayout extends React.Component<Props, State> {
     return null;
   }
 
+  shouldComponentUpdate(nextProps: Props, nextState: State) {
+    return (
+      !isEqual(
```

Comment:
I think we can revert this logic and use lodash.omit for removing properties which we won't compare. Like this:

```typescript
!isEqual(omit(this.props, propsToExclude), omit(nextProps, propsToExclude))
```

I suggested to revert this logic because it seems like excluded props list will be shorter than included one. But if I'm wrong, you can use your function `iclude`, but i think we should rename it into `pick`

Reply:
Function renamed to "pick", As for me, it’s better to include only parameters you need